lib/features/base/base_mailbox_controller.dart

```diff
@@ -418,10 +416,8 @@ abstract class BaseMailboxController extends BaseController {
         ..onConfirmAction(AppLocalizations.of(context).delete, () => onDeleteMailboxAction(presentationMailbox))
       ).show();
     } else {
-      showDialog(
-        context: context,
-        barrierColor: AppColor.colorDefaultCupertinoActionSheet,
-        builder: (context) => PointerInterceptor(
+      Get.dialog(
```

Comment:
what is the different? no context? it is safe to manage dialog?

Reply:
- Removing that Context is a special feature of GetX, it will not be affected, allowing us to create it in one place in the app.
- We use GetX's dialog to conveniently check if the dialog is displayed. Because GetX supports the `Get.isDialogOpen` function